cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java

```diff
@@ -141,13 +148,19 @@ private void compile(@CommandLine.Option(names = {"-s", "--source"}) String sour
             FileUtils.deleteDirectory(targetWorkspace.toFile());
             Files.createDirectory(targetWorkspace);
 
-            Translator<Database, Database> translator = TranslatorFactory
-                    .translator(source.getDbType(), target.getDbType());
+            if (source.getDbType().equals(target.getDbType())) {
+                log.info("Skipping translation because the target ({}) and source ({}) db types are the same.",
+                        target.getDbType(), source.getDbType());
+                translatedModels = getDatabases(sourceWorkspace).collect(Collectors.toList());
+            } else {
+                Translator<Database, Database> translator = TranslatorFactory
```

Comment:
this section looks like duplicate - can we put it on separate method? otherwise ltgm